libraries/SRV_Channel/SRV_Channel.cpp

```diff
@@ -202,3 +202,6 @@ uint16_t SRV_Channel::get_limit_pwm(LimitValue limit) const
     }
 }
 
+void SRV_Channel::set_output_unlimited_once() {
+        limited_pwm_mask &= (0U << ch_num);
```

Comment:
Indentation wrong.

This doesn't work. Zero left-shifted whatever number of times is still zero :slightly_smiling_face: It needs to be `limited_pwm_mask &= ~(1U << ch_num);`